.travis.yml

```diff
@@ -21,4 +21,5 @@ env:
   matrix:
     - CI_ACTION="npm run lint"
     - CI_ACTION="npm test"
+    - CI_ACTION="npm run-script test-integration"
```

Comment:
For clarity we should break the test into categories:

- `test-lint`
- `test-unit`
- `test-integration`